scripts/layers/layers.json

```diff
@@ -16,5 +16,41 @@
     "ap-southeast-1": "arn:aws:lambda:ap-southeast-1:632417926021:layer:webiny-v4-sharp:1",
     "ap-southeast-2": "arn:aws:lambda:ap-southeast-2:632417926021:layer:webiny-v4-sharp:1",
     "sa-east-1": "arn:aws:lambda:sa-east-1:632417926021:layer:webiny-v4-sharp:1"
+  },
+  "shelf-io-chrome-aws-lambda-layer": {
+    "ap-northeast-1": "arn:aws:lambda:ap-northeast-1:764866452798:layer:chrome-aws-lambda:20",
+    "ap-northeast-2": "arn:aws:lambda:ap-northeast-2:764866452798:layer:chrome-aws-lambda:20",
+    "ap-south-1": "arn:aws:lambda:ap-south-1:764866452798:layer:chrome-aws-lambda:20",
+    "ap-southeast-1": "arn:aws:lambda:ap-southeast-1:764866452798:layer:chrome-aws-lambda:20",
+    "ap-southeast-2": "arn:aws:lambda:ap-southeast-2:764866452798:layer:chrome-aws-lambda:20",
+    "ca-central-1": "arn:aws:lambda:ca-central-1:764866452798:layer:chrome-aws-lambda:20",
+    "eu-north-1": "arn:aws:lambda:eu-north-1:764866452798:layer:chrome-aws-lambda:20",
+    "eu-central-1": "arn:aws:lambda:eu-central-1:764866452798:layer:chrome-aws-lambda:20",
+    "eu-west-1": "arn:aws:lambda:eu-west-1:764866452798:layer:chrome-aws-lambda:20",
+    "eu-west-2": "arn:aws:lambda:eu-west-2:764866452798:layer:chrome-aws-lambda:20",
+    "eu-west-3": "arn:aws:lambda:eu-west-3:764866452798:layer:chrome-aws-lambda:20",
+    "sa-east-1": "arn:aws:lambda:sa-east-1:764866452798:layer:chrome-aws-lambda:20",
+    "us-east-1": "arn:aws:lambda:us-east-1:764866452798:layer:chrome-aws-lambda:22",
+    "us-east-2": "arn:aws:lambda:us-east-2:764866452798:layer:chrome-aws-lambda:20",
+    "us-west-1": "arn:aws:lambda:us-west-1:764866452798:layer:chrome-aws-lambda:20",
+    "us-west-2": "arn:aws:lambda:us-west-2:764866452798:layer:chrome-aws-lambda:20"
+  },
+  "sharp": {
+    "us-east-1": "arn:aws:lambda:us-east-1:632417926021:layer:sharp:10",
```

Comment:
Should this be layer `10` or `1`?

Reply:
`10` is correct. The thing with layers is that they preserve the version number for a specific layer name.
So if we used `sharp` in the past, in that region, and we published 9 versions, then deleted the layer, and then created a new layer with the same name, it will be version 10, not version 1. Because layers are identified by name, not some random ID.